sdk/servicebus/azure-servicebus/azure/servicebus/aio/_base_handler_async.py

```diff
@@ -147,9 +147,12 @@ def __init__(
         else:
             self._credential = credential # type: ignore
         self._container_id = CONTAINER_PREFIX + str(uuid.uuid4())[:8]
+        # overriding default 0.2 for async
```

Comment:
should we explain more in the comment here about reason for overriding in case someone goes looking?

Reply:
tbh my only reason is b/c this was already working for 1 for async and I didn't want to touch it b/c it's working